web/html/links-ii/image-mailto.md

```diff
@@ -105,3 +146,13 @@ Email Me
 - mail:
 - address:
 - email:
+
+---
+## Footnotes
+
+[1:%20 instead of space in a URL]
+
+The reason for replacing spaces with `%20` is because URLs have strict syntactic rules. URLs can't have spaces, so the ASCII character for space should be inserted instead. The `%` character, followed by two hexadecimal digits, is used to replace unsafe characters from a URL. The ASCII character for space is `20`. Making `%20` the replacement character for space.
```

Comment:
we can split this paragraph into multiple smaller ones to make it a bit more readable

```suggestion
The reason for replacing spaces with `%20` is because URLs have strict syntactic rules. 

URLs can't have spaces, so the ASCII character for space should be inserted instead. 

The `%` character, followed by two hexadecimal digits, is used to replace unsafe characters from a URL. 

The ASCII character for space is `20`, making `%20` the replacement character for space.
```